src/codewhisperer/service/transformationResultsViewProvider.ts

```diff
@@ -361,6 +364,18 @@ export class ProposedTransformationExplorer {
                 transformByQState.setSummaryFilePath(
                     path.join(pathContainingArchive, ExportResultArchiveStructure.PathToSummary)
                 )
+                transformByQState.setResultArchiveFilePath(pathContainingArchive)
+
+                // This metric is only emitted when placed before showInformationMessage
+                telemetry.codeTransform_vcsDiffViewerVisible.emit({
+                    codeTransformSessionId: codeTransformTelemetryState.getSessionId(),
+                    codeTransformJobId: transformByQState.getJobId(),
+                    result: MetadataResult.Pass,
+                })
+
+                // Do not await this so that the summary reveals without user needing to close this notification
+                void vscode.window.showInformationMessage(CodeWhispererConstants.viewProposedChangesMessage)
+                await vscode.commands.executeCommand('aws.amazonq.transformationHub.summary.reveal')
```

Comment:
Moved this code inside the `try` block since before it came after the `finally` block and it would always execute, even when `parseDiff` throws an error -- we only want to show the user the notification that their proposed changes are ready if no errors are experienced.